docs/features.md

```diff
@@ -163,3 +163,12 @@
 ##### steps:
 
 * TBD
+
+## Contracts
+### As a user, I want to be able to create a wallet so that I can store REV in it
+#### Create wallet
+##### test: test/test_wallet.py::test_create_wallet
+##### steps: 
+* send a message  to the process registered as 'rho:mint' with the structure **("create", pk, returnChannel)** where pk is the public key of the wallet
```

Comment:
The mint can't go in the registry. If it did, then anyone in the world could mint REV.

What's registered is a contract for making empty rev purses.

In the current implementation, there's an indirection via SystemInstancesRegistry, which is a little tedious. but it works.

https://github.com/rchain/rchain/blob/a62dd9b443f00104277841ae34c7684c862fe496/casper/src/main/rholang/SystemInstancesRegistry.rho

See for example:
https://github.com/dckc/RChain-API/blob/cli/examples/payment/aliceBobWallets.rho d1bad4e